src/include/simeng/BTB_BWTPredictor.hh

```diff
@@ -1,71 +1,73 @@
-#pragma once
```

Comment:
Everything in this file seems to be commented out? If not used then should the file just be deleted?